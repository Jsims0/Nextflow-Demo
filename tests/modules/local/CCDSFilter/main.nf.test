nextflow_process {

    name "Test Process CCDSFilter"
    script "modules/local/CCDSFilter/main.nf"
    process "CCDSFilter"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                input[0] = file("tests/testData/body.vcf")
                """
            }
        }

        then {
            assert process.success
            //assert snapshot(process.out).match()
        }

    }

}
