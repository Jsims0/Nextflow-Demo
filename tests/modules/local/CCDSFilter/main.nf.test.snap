{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "/Users/simonmurray/Documents/Nextflow-Demo/.nf-test/tests/5f10aa43fb0dbc0440c01fc790c9265d/work/68/647b1b970d060528db97a78164b5e0/CCDS.vcf"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.2"
        },
        "timestamp": "2024-06-12T19:55:57.445556"
    }
}